File: Torneio_beeeecrowd/1929.py
a,b,c,d = map(int,input().split())
if(a+b>c and b+c>a and c+a>b):
  print("S")
elif (a+b>d and b+d>a and d+a>b):
  print("S")
elif (a+d>c and d+c>a and c+a>d):
  print("S")
elif (d+b>c and b+c>d and c+d>b):
  print("S")
else:
  print("N")

File: Beecrowd/TELA_6/1865_1963/1929.cs
using System;
class Program{
  static void Main(string[] args){
    string[] v = Console.ReadLine().Split(' ');
    int p = int.Parse(v[0]);
    int q = int.Parse(v[1]);
    int r = int.Parse(v[2]);
    int s = int.Parse(v[2]);
      if(p + q >= r && q + r >= p && r + p >= q){
        Console.WriteLine("S");
      }
      else if(p + q >= s && q + s >= p && s + p >= q){
        Console.WriteLine("S");
      }
      else if(p + s >= r && s + r >= p && r + p >= s){
        Console.WriteLine("S");
      }
      else if(s + q >= r && q + r >= s && r + s >= q){
        Console.WriteLine("S");
      }
      else{
      Console.WriteLine("N");
    }
  }
}